backend/__tests__/setup.ts

```diff
@@ -1,4 +1,5 @@
-// Example
+import { config } from "dotenv";
+config(); // Get environment variables
```

Comment:
Put this after imports

Reply:
No I need it that way because```config()``` get the environment variables that are used by next imports.

Reply:
ok